lib/measures/tbd/resources/psi.rb

```diff
@@ -924,17 +915,19 @@ def processTBD(os_model, psi_set, io_path = nil, schema_path = nil)
     next unless deratable
 
     psi = {} # edge-specific PSI types
+    p = io[:unit].first[:psi]                                 # default unit PSI
 
     match = false
-    match = true if edge.has_key?(:io_type) # customized edge in TBD JSON file
-
-    t = edge[:io_type]        if match
-    p = io[:unit].first[:psi]                                     # default PSI
-    p = edge[:io_set]         if match && edge.has_key?(:io_set)  # custom PSI
-    psi[t] = io_p.set[p][t]   if match && io_p.set[p][t]
+    if edge.has_key?(:io_type) # customized edge in TBD JSON file
+      match = true
+      t = edge[:io_type]
+      p = edge[:io_set]       if edge.has_key?(:io_set)
+      edge[:set] = p          if io_p.set.has_key?(p)
+      psi[t] = io_p.set[p][t] if io_p.set[p].has_key?(t)
+    end
 
```

Comment:
No actual change in the end, but this took many forms during the debugging process.